@ndlib/gatsby-theme-marble/src/i18n/config.js

```diff
@@ -0,0 +1,25 @@
+import merge from 'lodash.merge'
+import defaultCommonJSON from 'assets/i18n/json/common.default.en'
+import commonJSON from 'assets/i18n/json/common.en'
+import defaultTextJSON from 'assets/i18n/json/text.default.en'
+import textJSON from 'assets/i18n/json/text.en'
+export default {
+  lng: 'en',
+  fallbackLng: 'en',
+  resources: {
+    en: {
+      common: merge({}, defaultCommonJSON, commonJSON),
+      text: merge({}, defaultTextJSON, textJSON),
```

Comment:
We have two namespaces here, each are loading from two json files merged together. This is so that we can have a "default" set of translations and an overwritable file that can be accessed through the shadow function pattern from Gatsby Themes.